packages/components/accordion/Accordion.mdx

```diff
@@ -1,36 +1,40 @@
 ---
 title: 'Accordion'
-type: 'component'
 slug: /components/accordion/
-github: 'https://github.com/contentful/forma-36/tree/master/packages/components/accordion'
-storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-accordion--basic'
+github: 'https://github.com/contentful/forma-36/tree/forma-v4/packages/components/accordion'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-accordion'
 typescript: ./src/Accordion.tsx,./src/AccordionItem/AccordionItem.tsx
+v4Doc: true
```

Comment:
Maybe we could have a status added and display the badge?

Reply:
Uhmmmm @domarku removed the badge from the design
(you can check the design in the Jira story, I'm not pasting it here since this repo is public, ping me if you need anything)

That's why I removed it from here

Reply:
Good catch, @burakukula. Still want to see if there's a better way to communicate status.